charts/soroban-rpc/pubnet-values.yaml

```diff
@@ -0,0 +1,224 @@
+global:
+  ## String to use to explicitly set namespace name in manifests.
+  ## Useful for those using "helm template" to render templates
+  # namespace: mynamespace
+
+  ## String to partially override common.fullname template (will maintain the release name)
+  ##
+  # nameOverride:
+
+  ## String to fully override common.fullname template
+  ##
+  # fullnameOverride:
+
+  ## List of pre-existing image Pull Secrets to use
+  # imagePullSecrets:
+  # - imagepullsecret1
+  # - imagepullsecret2
+
+  image:
+    sorobanRpc:
+      registry: docker.io
+      repository: stellar/soroban-rpc
+      tag: latest
+      ## Optional tag. 
+      ## If absent, defaults to the appVersion from Chart.yaml
+
+      ## Specify a imagePullPolicy
+      ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+      ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+      ##
+      pullPolicy: Always
+
+sorobanRpc:
+  ## See global.image.sorobanRpc for image settings
+
+  ## Soroban-rpc config variables
+  sorobanRpcConfig:
+    friendbotUrl: "https://friendbot.stellar.org/"
+    networkPassphrase: "Public Global Stellar Network ; September 2015"
+    historyArchiveUrls: "https://stellar-history-usw.franklintempleton.com/azuswshf401,https://stellar-archive-1-lobstr.s3.amazonaws.com,https://hercules-history.publicnode.org,https://stellar-full-history1.bdnodes.net,https://history.stellar.org/prd/core-live/core_live_003,https://history.stellar.org/prd/core-live/core_live_002,https://history.stellar.org/prd/core-live/core_live_001"
```

Comment:
Is the `azuswshf401` correct? or a typo?

Reply:
yes, it looks correct, was copied from the stellar-core.cfg on stg-pubnet